src/index.html

```diff
@@ -1,16 +1,42 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Search Form Example</title>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-form search-form_big" data-qa="big">
+      <form action="#" method="GET">
+        <div class="search-form__container">
+          <label class="search-form__label search-form__label_big">
+            <input type="text" 
+            class="search-form__input search-form__input_big" 
+            placeholder="Try &quot;Los Angeles&quot;" 
+            data-qa="keypress"
+            >
+            <button type="submit" class="search-form__button">
+            </button>
+          </label>
+        </div>
+      </form>
+    </div>
+
+    <div class="search-form search-form_small" data-qa="small">
```

Comment:
Same comments go for this form.
It's generally recommended to keep the HTML structure as simple and concise as possible, using wrappers and containers only when necessary to achieve the desired functionality and styling. Using redundant wrappers is not recommended because it can lead to unnecessary complexity and bloated markup.
So, it'll be enough to use this structure for each form
```
<form>
  <label>
    <input>
  </label>
</form>